recipes/stc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_property(TARGET ${PROJECT_NAME} PROPERTY C_STANDARD 11)
```

Comment:
The readme and description say `C99`

Reply:
yes, this is to fix:
```
error: 'for' loop initial declarations are only allowed in C99 or C11 mode
```
it could be either C99 or C11